src/lib/detectInstall.js

```diff
@@ -32,25 +32,27 @@ function detectInstall (context) {
 
   // the plugin we're trying to install
   let plugin = parameters.second
+  let packageName = plugin.split('@')[0]
```

Comment:
Just a heads up.  I know I lost the argument to allow non-ignite prefixes, but this is a common thing nowadays to use scoped packages.  They look like `@infinitered/ignite`.  That `@` will mess with this code you've written here.  Maybe we could just do a quick bailout if it `startsWith` `@`?

That's bonus points, though.  I'm ok with this PR as is.